packages/ui-markdown-editor/src/utilities/toolbarHelpers.js

```diff
@@ -91,17 +89,26 @@ export const insertThematicBreak = (editor, type) => {
   Transforms.insertNodes(editor, tBreakNode);
 };
 
-export const insertLinebreak = (editor, type) => {
+export const insertSoftBreak = (editor, type) => {
   const text = { object: 'text', text: '' };
   const br = { type, children: [text], data: {} };
   Transforms.insertNodes(editor, br);
   Transforms.move(editor, { distance: 1, unit: 'character' });
 };
 
-export const insertHeadingbreak = (editor) => {
+export const insertLineBreak = (editor) => {
+  const text = { object: 'text', text: '' };
+  const lineBreakNode = { type: LINEBREAK, children: [text] };
+  Transforms.insertNodes(editor, lineBreakNode);
+  Transforms.move(editor, { distance: 1, unit: 'character' })
+  return;
+}
+
+export const insertHeadingBreak = (editor) => {
   const text = { object: 'text', text: '' };
   const n = { object: "block", type: 'paragraph', children: [text] };
   Transforms.insertNodes(editor, n);
+  Transforms.move(editor, { distance: 1, unit: 'character' })
```

Comment:
I ran this locally and actually I think it could be this line that we may want to back out.